CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+**Thanks**
```

Comment:
Nit: The `Thanks` section is usually located below `Features` and `Bugfixes` (i.e., as in the 2.9.0 section https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md).